packages/calcite-components/src/components/handle/assets/handle/t9n/messages.json

```diff
@@ -1,5 +1,5 @@
 {
-  "dragHandle": "Drag handle",
+  "dragHandle": "Reposition or move: {itemLabel}",
```

Comment:
Should we choose one of “Reposition” or “move” here?

Reply:
We could just go with reposition. I was just thinking of sort vs move out